test/controllers/profiles_controller_test.rb

```diff
@@ -4,68 +4,103 @@ class ProfilesControllerTest < ActionDispatch::IntegrationTest
   include Devise::Test::IntegrationHelpers
 
   setup do
-    @user = users(:one)
-    @user.admin = true
-    @user1 = users(:two)
-    @profile1 = profiles(:one)
+    @bob_attributes = {
+      first_name: "Bob",
+      last_name: "Bobman",
+      nickname: "Bobby",
+      email: "bob@bob.com",
+      address: "12345 High St",
+      neighborhood: "Neighborhood One",
+      spouse: "Jim",
+      cell: "1234567",
+      landline: "123456789",
+    }
   end
 
-  test "should get new" do
-    login_as(@user)
-    get profiles_new_url
+  test "should show all profiles with no search" do
+    login_as(users(:normal))
+    get profiles_path
     assert_response :success
+    assert_select ".profile" do |profile_elements|
+      assert_select profile_elements[0], ".profile__name", "Green, Frog"
```

Comment:
`Green, Frog` feels iffy, consider changing it to (profiles(:frog)).last_name + .first_name

Reply:
I'm keeping the string literal because it is simpler and no less maintainable.